egs/rm/s5/local/chain/tuning/run_tdnn_wsj_rm_1b.sh

```diff
@@ -0,0 +1,208 @@
+#!/bin/bash
+# _1b is as _1a, but different as follows
+# 1) uses wsj phone set phones.txt and new lexicon generated using word pronunciation
+#    in swj lexincon.txt and rm words not presented in wsj are added as oov
+#    in new lexicon.txt.
+# 2) It uses wsj tree-dir and generates new rm alignments and lattices using
+#    wsj gmm model.
+# 3) It also train phone LM using weighted combination of alignemts from wsj
+#    and rm, which is used in chain denominator graph.
+#    Since we use phone.txt from source dataset, this can be helpful in cases
+#    where there is few training data in target and some 4-gram phone sequences
+#    have no count in target.
+# 4) It does not replace the output layer from already-trained model with new
+#    randomely initialized output layer and and re-train it using target dataset.
+
+
+# This script uses weight transfer as Transfer learning method
+# and use already trained model on wsj and fine-tune the whole network using rm data
+# while training the last layer with higher learning-rate.
+# The chain config is as run_tdnn_5n.sh and the result is:
+# System tdnn_5n tdnn_wsj_rm_1a tdnn_wsj_rm_1b tdnn_wsj_rm_1c
+# WER      2.71     2.09            3.45          3.38
+set -e
+
+# configs for 'chain'
+stage=7
+train_stage=-4
+get_egs_stage=-10
+
+# training options
+num_epochs=2
+initial_effective_lrate=0.005
+final_effective_lrate=0.0005
+leftmost_questions_truncate=-1
+max_param_change=2.0
+final_layer_normalize_target=0.5
```

Comment:
This option is not used in this script. Check other options like leftmost_questions_truncate is not used in _b and _c. You can hard code some of the options directly instead of allowing the used to control is. e.g. frames_per_eg is always going to be 150.